vimeo__player/@vimeo-player-tests.ts

```diff
@@ -0,0 +1,456 @@
+import { Player } from '@vimeo/player'
```

Comment:
sorry for confusion, just to be sure: 
only the `vimeo__player-tests.ts` should be in the `types/vimeo__player` folder? 
the `vimeo__player/index.d.ts` file stays in the main folder? It was suggested in a "contributor handbook", so I'm unsure whats the right way.